pkg/daemon/ceph/agent/flexvolume/types.go

```diff
@@ -48,6 +48,7 @@ type AttachOptions struct {
 	Path             string `json:"path"` // Path within the CephFS to mount
 	RW               string `json:"kubernetes.io/readwrite"`
 	FsType           string `json:"kubernetes.io/fsType"`
+	FsGroup          string `json:"kubernetes.io/fsGroup"`
```

Comment:
From where does this get set? By Kubernetes?

Reply:
IIRC it is passed in by Kubernetes, but is currently being ignored in Rook's flexvolume driver. This change should pick it up properly.